scripts/grab_ss.py

```diff
@@ -0,0 +1,77 @@
+from Xlib import display
+import numpy as np
+from PIL import Image, ImageGrab
+import subprocess
+import time
+
+def find_window_id(window_name):
```

Comment:
What is this grab ss file ? Is it Linux ? Why is it named differently ?

Reply:
It was for Linux. It is named differently because it is just meant to grab ss and store in a folder. It is meant to be a throwaway script.